docs/content/system-arguments.md

```diff
@@ -55,57 +55,57 @@ System arguments include most HTML attributes. For example:
 | `border_radius` | Integer | One of `0`, `1`, `2`, or `3`. |
 | `border_right` | Integer | Set to `0` to remove the right border. |
 | `border_top` | Integer | Set to `0` to remove the top border. |
-| `border` | Symbol | One of `:bottom`, `:left`, `:right`, `:top`, `:x`, `:y`, or `true`. |
-| `box_shadow` | Boolean, Symbol | Box shadow. One of `:extra_large`, `:large`, `:medium`, `:none`, or `true`. |
+| `border` | Symbol | One of `:left`, `:top`, `:bottom`, `:right`, `:y`, `:x`, or `true`. |
+| `box_shadow` | Boolean, Symbol | Box shadow. One of `:medium`, `:large`, `:extra_large`, `:none`, or `true`. |
 
 ## Color
 
 | Name | Type | Description |
 | :- | :- | :- |
-| `bg` | String, Symbol | Background color. Accepts either a hex value as a String or one of `:canvas`, `:canvas_inset`, `:canvas_inverse`, `:danger`, `:danger_inverse`, `:info`, `:info_inverse`, `:overlay`, `:primary`, `:secondary`, `:success`, `:success_inverse`, `:tertiary`, `:warning`, or `:warning_inverse`. |
-| `border_color` | Symbol | Border color. One of `:danger`, `:info`, `:inverse`, `:primary`, `:secondary`, `:success`, `:tertiary`, or `:warning`. |
-| `color` | Symbol | Text color. One of `:icon_danger`, `:icon_info`, `:icon_primary`, `:icon_secondary`, `:icon_success`, `:icon_tertiary`, `:icon_warning`, `:text_danger`, `:text_inverse`, `:text_link`, `:text_primary`, `:text_secondary`, `:text_success`, `:text_tertiary`, `:text_warning`, or `:text_white`. |
+| `bg` | String, Symbol | Background color. Accepts either a hex value as a String or one of `:primary`, `:secondary`, `:tertiary`, `:canvas`, `:canvas_inset`, `:canvas_inverse`, `:info`, `:info_inverse`, `:success`, `:success_inverse`, `:warning`, `:warning_inverse`, `:danger`, `:danger_inverse`, or `:overlay`. |
+| `border_color` | Symbol | Border color. One of `:primary`, `:secondary`, `:tertiary`, `:info`, `:success`, `:warning`, `:danger`, or `:inverse`. |
+| `color` | Symbol | Text color. One of `:icon_primary`, `:icon_secondary`, `:icon_tertiary`, `:icon_info`, `:icon_success`, `:icon_warning`, `:icon_danger`, `:text_primary`, `:text_secondary`, `:text_tertiary`, `:text_link`, `:text_success`, `:text_warning`, `:text_danger`, `:text_white`, or `:text_inverse`. |
 
 ## Flex
 
 | Name | Type | Description |
 | :- | :- | :- |
-| `align_items` | Symbol | One of `:baseline`, `:center`, `:flex_end`, `:flex_start`, or `:stretch`. |
-| `align_self` | Symbol | One of `:auto`, `:baseline`, `:center`, `:end`, `:start`, or `:stretch`. |
+| `align_items` | Symbol | One of `:flex_start`, `:flex_end`, `:center`, `:baseline`, or `:stretch`. |
```

Comment:
Do you think we should change the way we're sorting these arguments?

Reply:
Definitely not, thanks for catching this! As @jonrohan mentioned, this happened as a result of changing the `one_of` helper. Should be fixed now.